### ISSUE TITLE
All Overloading Functions inside the Package are not Extracted .

### ISSUE THREAD

**Issue by jagmohankaintura-tl** [darold/ora2pg#985]
While doing extraction for Packages, if we have multiple overloaded functions present in the Package Body only the last function   coming in sequence of extraction is getting extracted  and rest others are not part of extraction.

Package functions are  been extracted   into specific files with naming convention as :

<package_name>.<function_name>_package.sql. 

I think as we generate single files for specific function, when we have multiple functions with same name they are getting over-written in same file and only function getting extracted last with same name  is getting placed in the file.

Is there any possibility if we change method from replace mode to append mode. this way we may have multiple functions with same name extracted in same file.

**Comment by naveenjul29** [comment]
I am second on this, me too got the same problem. only the last method(overloaded) got exported inside the package.

**Comment by darold** [comment]
Yes this is the problem when using hash to store the definition with a key based on package name and function name, the last parsed overloaded function erase all other definitions. It is in my todo list.

**Comment by darold** [comment]
Ora2pg needs to be patched to support overloaded function, this is a huge patch that requires time.
